environments/.env.production

```
@@ -1 +1,3 @@
 NEXT_PUBLIC_GITHUB_URL=https://api.github.com/
+NEXT_PUBLIC_POSTHOG_KEY=phc_s5dpMW7DvBDzwFpo0dWrj59N2iFi6aj59yn4bW8gmP8
+NEXT_PUBLIC_POSTHOG_HOST=https://us.i.posthog.com
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)